qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -201,20 +199,23 @@ def _to_instruction(
 
         return super()._to_instruction(elem, basis_gates)
 
-    def __set_up_interleaved_op(self, basis_gates: Optional[Tuple[str, ...]]) -> None:
+    def __set_up_interleaved_op(self) -> None:
         # Convert interleaved element to transpiled circuit operation and store it for speed
+        basis_gates = self._get_basis_gates()
         # Convert interleaved element to circuit
+        if isinstance(self._interleaved_op, Clifford):
+            self._interleaved_op = self._interleaved_op.to_circuit()
+
+        # assert isinstance(self._interleaved_op, (Instruction, QuantumCircuit))
         if isinstance(self._interleaved_op, QuantumCircuit):
             interleaved_circ = self._interleaved_op
-        elif isinstance(self._interleaved_op, Clifford):
-            interleaved_circ = self._interleaved_op.to_circuit()
         elif isinstance(self._interleaved_op, Gate):
             interleaved_circ = QuantumCircuit(self.num_qubits, name=self._interleaved_op.name)
             interleaved_circ.append(self._interleaved_op, list(range(self.num_qubits)))
```

Comment:
For interleaved element, it should be cared by `transpile` called a few lines later. However, I have recently realized a bug related with the direction of basis gates. In BackendV2, all gates are direction aware so we need to handle all basis gates in direction-aware as you pointed out. It may be a critical bug in the era of BackendV2. I'll address the issue in a separate PR.